signature/policy_eval_sigstore_test.go

```diff
@@ -320,13 +348,18 @@ func TestPRrSigstoreSignedIsSignatureAccepted(t *testing.T) {
 
 	// Successful key+Rekor use
 	pr, err = newPRSigstoreSigned(
-		PRSigstoreSignedWithKeyPath("fixtures/cosign2.pub"),
+		PRSigstoreSignedWithKeyPaths([]string{"fixtures/cosign.pub", "fixtures/cosign2.pub"}),
 		PRSigstoreSignedWithRekorPublicKeyPath("fixtures/rekor.pub"),
 		PRSigstoreSignedWithSignedIdentity(prm),
 	)
+
 	require.NoError(t, err)
 	sar, err = pr.isSignatureAccepted(context.Background(), testKeyRekorImage, testKeyRekorImageSig)
+	require.NoError(t, err)
```

Comment:
Isn’t this redundant with the `assertAccepted` below? Or, alternatively, if `assertAcepted` now depends on `err == nil` to do other checks, this `require` should happen inside `assertAccepted`.